packages/synapse-react-client/src/components/UserOrTeamBadge/UserOrTeamBadge.tsx

```diff
@@ -42,7 +40,7 @@ export default function UserOrTeamBadge(props: UserOrTeamBadgeProps) {
       'Expected one of principalId or userGroupHeader to be defined but both were null or undefined',
     )
     return <></>
-  } else if (userGroupHeader === undefined) {
+  } else if (userGroupHeader === undefined || isLoadingUserGroupHeader) {
```

Comment:
Include `isLoading` so that `UserOrTeamBadge` displays once the data is fetched, otherwise the skeleton remains if there is not another render -- see [this story](https://sage-bionetworks.github.io/synapse-web-monorepo/?path=/story/ui-userorteambadge--user)

Reply:
Remove this change -- the changes @nickgros made to the Storybook cache invalidation in #837 seem to have resolved this issue.